WrightTools/artists/_base.py

```diff
@@ -461,6 +462,73 @@ def imshow(self, *args, **kwargs):
             super().invert_yaxis()
         return out
 
+    def scatter(self, *args, **kwargs):
+        """Scatter plot a channel against two _variables_.
+        Scatter point color reflects channel values.
+        Data need not be structured.
+        If data object is not provided, scatter reverts to the [matplotlib parent method](https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.scatter.html?highlight=scatter#matplotlib.axes.Axes.scatter)
+
+        Parameters
+        ----------
+        data : 2D WrightTools.data.Data object
+            Data to plot.
+        x : int or string (optional)
+            variable name or index for x (abscissa) axis.  Default is 0
+        y : int or string (optional)
+            variable name or index for y (ordinate) axis.  Default is 1
+        channel : int or string (optional)
+            Channel index or name. Default is 0.
+        autolabel : {'none', 'both', 'x', 'y'}  (optional)
+            Parameterize application of labels directly from data object. Default is none.
+        xlabel : string (optional)
+            xlabel. Default is None.
+        ylabel : string (optional)
+            ylabel. Default is None.
+        **kwargs
+            matplotlib.axes.Axes.scatter__ optional keyword arguments.
+
+            __ https://matplotlib.org/api/_as_gen/matplotlib.pyplot.scatter.html
+
+        Returns
+        -------
+        matplotlib.collections.PathCollection
+        """
+        args = list(args)
+        if isinstance(args[0], Data):
+            data = args.pop(0)
+            x = kwargs.pop("x", 0)
+            x = wt_kit.get_index(data.variable_names, x)
+            x = data.variables[x].full.flatten()
+
+            y = kwargs.pop("y", 1)
+            y = wt_kit.get_index(data.variable_names, y)
+            y = data.variables[y].full.flatten()
+            args = [x, y] + args[2:]
```

Comment:
why args[2:]?

one element is removed by pop on line 498, so you are skipping two args